metadata.json

```diff
@@ -33,6 +33,7 @@
   ],
   "dependencies": [
     { "name": "puppetlabs/stdlib", "version_requirement": ">3.x" },
-    { "name": "puppetlabs/concat", "version_requirement": ">1.x" }
+    { "name": "puppetlabs/concat", "version_requirement": ">1.x" },
+    { "name": "kemra102/elrepo",   "version_requirement": ">1.x" }
```

Comment:
I've never actually used this module!  It seemed like the best choice of modules on the forge.  Needs testing.